FeedStoreChallenge/Infrastructure/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Fernando Goulart on 01/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
+	@NSManaged internal var feed: NSOrderedSet
```

Comment:
`internal` is a default access level, no need to declare it explicitly. Please remove it.

```suggestion
	@NSManaged var timestamp: Date
	@NSManaged var feed: NSOrderedSet
```